sinks/influxdb/driver.go

```diff
@@ -85,101 +94,83 @@ func (self *influxdbSink) metricToSeries(timeseries *sink_api.Timeseries) *influ
 	if timeseries.MetricDescriptor.Type.String() != "" {
 		seriesName = fmt.Sprintf("%s_%s", seriesName, timeseries.MetricDescriptor.Type.String())
 	}
-
-	// Add the real metric value.
-	columns = append(columns, "value")
-	values = append(values, timeseries.Point.Value)
+	point := influxdb.Point{
+		Measurement: seriesName,
+		Tags:        make(map[string]string, 0, len(timeseries.Point.Labels)),
+		Fields: map[string]interface{}{
```

Comment:
You're right: https://play.golang.org/ (press the format button)

It looks like gofmt treats composite literals differently.